authentication/src/main/java/io/camunda/authentication/converter/TokenClaimsConverter.java

```diff
@@ -74,14 +72,15 @@ public CamundaOAuthPrincipalServiceImpl(
     oidcGroupsLoader = new OidcGroupsLoader(groupsClaim);
   }
 
-  @Override
-  public OAuthContext loadOAuthContext(final Map<String, Object> claims)
-      throws OAuth2AuthenticationException {
-    final var authContextBuilder = new AuthenticationContextBuilder();
-    final var principals = oidcPrincipalLoader.load(claims);
+  public CamundaAuthentication convert(final Map<String, Object> tokenClaims) {
```

Comment:
Note: With https://github.com/camunda/camunda/issues/36040, I am planning to consolidate resolving the memberships independently, whether it is basic auth or OIDC. At this stage, the code "just" moved and remains duplicated (nothing changed in this regards)